nncf/quantization/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -268,6 +272,9 @@ def _create_input_data(self,
         :return: The dictionary of the blob by input name.
         """
         input_blob = self._backend_entity.create_blob(input_shape, input_fp, channel_axis)
+        if input_name is None:
+            # For unnamed inputs, as in pytorch
```

Comment:
In pytorch you have no way to get sub graph by names like onnx and ov 
https://github.com/openvinotoolkit/nncf/blob/cd4171c2cc09ffbdd0ae0520e9f19d2840fe6c44/nncf/onnx/graph/model_transformer.py#L371
Only get node node by name, for FBC it works correctly , as i understand the algorithm.
https://github.com/openvinotoolkit/nncf/blob/cd4171c2cc09ffbdd0ae0520e9f19d2840fe6c44/nncf/quantization/algorithms/fast_bias_correction/algorithm.py#L228-L229

Reply:
Please, provide the output graph. 
Also, the `ModelExctactionCommand` should be applicable for `BiasCorrection`. Thus, we need to have a general command in case of multiple inputs.

Reply:
If export pt node to onnx graph (input/output nodes from onnx, not available in pt), extracted graphs looks like:
![image](https://user-images.githubusercontent.com/48012821/229947639-bafd9e2e-3d8e-469d-b06b-44a41615473c.png)

General command it will be hard to implement. PT model is not a graph, node have no names you have no any function to extract subgraphs by names as in onnx.

Reply:
Can we make this function backend specific? It will not lead to a if-clause. 
Moreover, now this function doesn't do much. It just wraps the return value to the correct type.

Reply:
Thank you for the graph. This is what I've asked about.
> General command it will be hard to implement.

So, what should we do for the `BiasCorrection` algorithm in PyTorch? Any suggestions?

Reply:
Also, as I can see, there exists the name of the sub-graph. 
Is the condition `if input_name is None:` really work? What would be in case we feed the named `input_data`?

Reply:
> Also, as I can see, there exists the name of the sub-graph.
It's exported graph by `torch.onnx.export(extracted_model, ...)`, and names in graphs on screenshot is default names for exported model. 

> If export pt node to onnx graph **(input/output nodes from onnx, not available in pt)**

Extracted model is instance of NNCFConv2d with prehook for FQ weights.